backend/infrahub/api/schema.py

```diff
@@ -247,7 +248,12 @@ async def load_schema(
         if not await permission_backend.has_permission(
             db=db,
             account_id=account_session.account_id,
-            permission=f"global:{GlobalPermissions.MANAGE_SCHEMA.value}:{PermissionDecision.ALLOW.value}",
+            permission=GlobalPermission(
+                id="",
+                name="",
+                action=GlobalPermissions.MANAGE_SCHEMA.value,
+                decision=PermissionDecision.ALLOW_ALL.value,
```

Comment:
In the end I think we'd want to check the `branch` variable to see we need the ALLOW_DEFAULT/ALLOW_ALL or ALLOW_OTHER/ALLOW_ALL permission.